docs/docs/interactions.md

```diff
@@ -4,31 +4,31 @@ title: Cross handler interactions
 sidebar_label: Cross handler interactions
 ---
 
-Gesture handlers can "communicate" with each other in order to allow for defining complex gesture interactions and to control how they [activate](state.md#active) in certain scenarios.
+Gesture handlers can "communicate" with each other to support complex gestures and control how they *[activate](state.md#active)* in certain scenarios.
 
-There are two means of such a control at the moment described in the sections below.
-In each of the cases it is necessary to provide a reference of one handler as a property to the other.
+There are two means of momentary gesture control described in the sections below.
+In each case, it is necessary to provide a reference of one handler as a property to the other.
 Gesture handler relies on ref objects created using [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html) and introduced in [React 16.3](https://reactjs.org/blog/2018/03/29/react-v-16-3.html#createref-api).
 
 ## Simultaneous recognition
 
-By default at a given time only one gesture handler is allowed to be in an [active](state.md#active) state.
-So when a gesture handler recognizes a gesture, it [cancels](state.md#cancelled) all the other handlers in [began](state.md#began) and prevents any new handlers from receiving a stream of touch events for as long as it is [active](state.md#active).
+By defauly, only one gesture handler is allowed to be in the [`ACTIVE`](state.md#active) state.
+So when a gesture handler recognizes a gesture, or becomes [`ACTIVE`](state.md#active), it [cancels](state.md#cancelled) all other handlers in the [`BEGAN`](state.md#began) state *and* prevents any new handlers from receiving a stream of touch events as long as it remains [`ACTIVE`](state.md#active).
```

Comment:
", or becomes [`ACTIVE`](state.md#active),"

Did you test that? I think recognizing a gesture (that is going into a BEGAN state) is sufficient.

Reply:
"*and*" 

Not sure if the emphasis is needed here.

Reply:
so the ", or becomes ACTIVE," is reiterating what's said in the "Handler State" section that a handle is active when it "...has recognized a gesture..."
So I'm not quite sure I understand your suggestion.

Right. No emphasis needed.

Reply:
Question: 
Does the `BEGAN` state include both actions: 
"receiving stream of touch events" and "recognizing the gesture"?

Reply:
AFAIK began only refers to the first one, recognizing the gesture should fire the ACTIVE event.

In that case, I'm wrong in the first comment. Recognizing a gesture is equivalent to getting into an ACTIVE state (see [docs](https://docs.swmansion.com/react-native-gesture-handler/docs/state#active)) so ", or becomes ACTIVE," is redundant here (You can change it to something like "So when a gesture handler recognizes a gesture (getting into [`ACTIVE`](state.md#active) state)" if you want to include the name of the state here).

Reply:
Actually, I think it's just best to go back to the way it was and leave the extra sentence out. I'll fix it. Plz let me know what you think.

Reply:
I'm fine with that, thanks.